readthedocs/doc_builder/config.py

```diff
@@ -0,0 +1,114 @@
+
+from readthedocs_build.config import (ConfigError, BuildConfig,
+                                      load as load_config)
+
+
+class ConfigWrapper(object):
+
+    """
+    A config object that wraps the Project & YAML based configs.
+
+    Gives precidence to YAML, falling back to project if it isn't defined.
+
+    We only currently implement a subset of the existing YAML config.
+    This should be the canonical source for our usage of the YAML files,
+    never accessing the config object directly.
+
+    """
+
+    def __init__(self, version, yaml_config):
+        self._version = version
+        self._project = version.project
+        self._yaml_config = yaml_config
+
+    @property
+    def install_project(self):
+        if 'setup_py_install' in self._yaml_config.get('python', {}):
```

Comment:
Sorry, it was apparent *what* is happening, just the importance of the `setup_py_install` configuration option wasn't clear. I assume this means install using setup.py, but that would be good to say explicitly. The prose docs on the option could be expanded at least, though linking ref -> prose wouldn't hurt